docker/data-science/notebooks/aips.py

```diff
@@ -32,6 +33,9 @@ def healthcheck():
   except:
       print ("Error! One or more containers are not responding.\nPlease follow the instructions in Appendix A.")
 
+def get_engine():
```

Comment:
Should we introduce engine name as a parameter from beginning ? With default equal to `solr` ?

Reply:
The factory pattern will work in the future when more engines are created. The implementation can be deferred